### PR TITLE
FOGL-1594 - Simplify omf and ocs plugin classes

### PR DESCRIPTION
**Pull request by amarendra-dianomic**
Clarity needed: Role of "type_id" is still not clear. DB schema says that it "identifies the specific PI Server type". What is meant by this? Architecture doc says that it is a kind of internal index maintained till duration of the connection and is to be used in packets to be sent. What is meant by that? If it identifies a PI server, then why this job cannot be done by the producerToken (and also by tenantId in case of OCS)?

### CODE REVIEW

File: python/foglamp/plugins/north/pi_server/pi_server.py [hunk 1]
```diff
@@ -148,115 +92,24 @@
         "description": "OMF format property to apply to the type Integer",
         "type": "string",
         "default": "int64"
+    },
+    "verifySSL": {
```
Comment (praveen-garg) [inline python/foglamp/plugins/north/pi_server/pi_server.py]: I hope this is being used to decide?!

File: python/foglamp/plugins/north/pi_server/pi_server.py [hunk 2]
```diff
@@ -148,115 +92,24 @@
...
+        "description": "Verify SSL certificate",
+        "type": "boolean",
+        "default": "false"
+    },
+    "compression": {
```
Comment (praveen-garg) [inline python/foglamp/plugins/north/pi_server/pi_server.py]: also use this in code, and send compression in message header to PI server
```
# Compress json  payload, if specified
        compression = 'none'
        if USE_COMPRESSION:
            msg_body = gzip.compress(bytes(json.dumps(reads_data), 'utf-8'))
            compression = 'gzip'
        else:
            msg_body = json.dumps(reads_d
```

File: python/foglamp/plugins/north/common/common.py [hunk 3]
```diff
@@ -4,155 +4,63 @@
...
-    """Evaluates and converts to the type in relation to its actual value, for example "180.2" to float 180.2
-
-     Args:
-        value : value to evaluate and convert
-     Returns:
-         value_converted: converted value
-     Raises:
-     """
-
+    """Evaluates and converts to the type in relation to its actual value, for example "180.2" to float 180.2"""
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/north/common/common.py]: Why are we removing useful comment information?

File: python/foglamp/plugins/north/common/common.py [hunk 4]
```diff
@@ -4,155 +4,63 @@
...
-    """Evaluates the type in relation to its value
-
-     Args:
-        value : value to evaluate
-     Returns:
-         Evaluated type {integer,number,string}
-     Raises:
-     """
-
+    """Evaluates the type in relation to its value"""
```
Comment (MarkRiddoch) [inline python/foglamp/plugins/north/common/common.py]: Please do not remove comments unless they are incorrect

**Comment by praveen-garg** [comment]
@MarkRiddoch  what is the best time, we have in plan to do it? or we should never do it?

**Comment by MarkRiddoch** [comment]
We should wait until the current batch of functionality changes is complete. It is more important to get the functionality we want and then address any refactoring we want to do. Trying to do both at the same time is wasteful of resource and error prone.

File: python/foglamp/plugins/north/common/omf_translator.py [hunk 5]
```diff
@@ -0,0 +1,502 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" The OMF North is a plugin output formatter for the FogLAMP appliance.
```
Comment (praveen-garg) [inline python/foglamp/plugins/north/common/omf_translator.py]: this will need a fix

File: python/foglamp/plugins/north/common/omf_translator.py [hunk 6]
```diff
@@ -0,0 +1,502 @@
...
+from foglamp.common.storage_client import payload_builder
+from foglamp.common.storage_client.exceptions import StorageServerError
+
+__author__ = "Stefano Simonelli, Amarendra K Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+class OmfType:
```
Comment (praveen-garg) [inline python/foglamp/plugins/north/common/omf_translator.py]: need a docstring

**Comment by MarkRiddoch** [comment]
I suspect so much has moved on we are best off withdrawing this PR

**Comment by praveen-garg** [comment]
Closing this PR, refactoring  is still required, but may be at later stage.

**Pull request closed without merging**